sample1.py

```diff
@@ -1,7 +1,10 @@
+import fileinput
+
 def inputNumbers():
-    numberListString = raw_input("Please input your numbers, separated by commas (and spaces): ")
```

Comment:
raw_input() helyett input() van python3-ban. De mindkettőnél jobb a fileinput library-t használni, ami azt csinálja hogy alapból stdin-ről olvas, de ha megadsz a programnak futtatáskor paraméterként egy fájlt, akkor azt olvassa helyette. (Bár olyan esetben ha csak 1 sort olvasol be az kicsit fura, de a legtöbb esetben az a jobb.)